index.css

```diff
@@ -246,26 +251,27 @@ main {
 .works {
   background-color: rgba(255, 255, 255, 1);
   display: flex;
-  flex-direction: row;
+  justify-content: space-between;
   flex-wrap: wrap;
-  justify-content: space-around;
+  flex-direction: column;
+  margin: 95px 0;
 }
 
 .grid {
   width: auto;
   height: auto;
   align-items: start;
-  margin-top: 5%;
 }
 
 .grid-item {
   display: inline-block;
   background-color: rgba(255, 255, 255, 1);
```

Comment:
Hi, @ibukun53 Please take a look at this, why don't we use this as a Variable, I mean Let's create a ::root and add all the colors in one place and use as a variable for understanding.